connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorSourceConnector.java

```diff
@@ -79,8 +80,8 @@ public class MirrorSourceConnector extends SourceConnector {
     private List<TopicPartition> knownTargetTopicPartitions = Collections.emptyList();
     private ReplicationPolicy replicationPolicy;
     private int replicationFactor;
-    private AdminClient sourceAdminClient;
-    private AdminClient targetAdminClient;
+    private ForwardingAdmin sourceAdminClient;
+    private ForwardingAdmin targetAdminClient;
```

Comment:
Again, there no need for this because the call sites don't care.

Reply:
Same as the others.